web/src/main/java/org/cbioportal/web/StudyViewController.java

```diff
@@ -845,6 +841,29 @@ public List<CaseListDataCount> fetchCaseListCounts(
                 .collect(Collectors.toList());
 
     }
+
+    @PreAuthorize("hasPermission(#involvedCancerStudies, 'Collection<CancerStudyId>', T(org.cbioportal.utils.security.AccessLevel).READ)")
+    @RequestMapping(value = "/genomic-data-counts/fetch", method = RequestMethod.POST,
```

Comment:
This has to support `StudyViewFilter`.  Look into `GenericAssayDataCountFilter` or `/generic-assay-data-counts/fetch` api